istio-control/istio-autoinject/files/injection-template.yaml

```diff
@@ -67,7 +67,7 @@ template: |
   {{- else }}
     image: "{{ .Values.global.hub }}/{{ .Values.global.proxy_init.image }}:{{ .Values.global.tag }}"
   {{- end }}
-    imagePullPolicy: IfNotPresent
+    imagePullPolicy: "{{ valueOrDefault .Values.global.imagePullPolicy `Always` }}"
```

Comment:
Looks like backquote is not valid here.

Reply:
That should be working. we have similar usage in this file.